app/src/main/java/wordtextcounter/details/main/util/RateUsHelper.kt

```diff
@@ -48,6 +54,24 @@ object RateUsHelper {
         }
       }
 
+      btLater.setOnClickListener {
+        onRemindLaterSelected(activity)
+        dialog.dismiss()
+      }
+
+      btNever.setOnClickListener {
+        onNoSelected(activity)
+        dialog.dismiss()
+      }
+
+      dialog.setOnKeyListener { _, keyCode, _ ->
+        if (keyCode == KEYCODE_BACK) {
+          onNoSelected(activity)
```

Comment:
Should we consider back press as never? Maybelater would be appropriate in this case I think